packages/sdk-core/package.json

```diff
@@ -35,9 +35,10 @@
         "fix:prettier": "yarn lint:prettier --write",
         "fix:eslint": "yarn lint:eslint --fix",
         "pre-commit": "run-s lint:*",
-        "generate-abi-files": "rm -rf src/abi && mkdir src/abi && tasks/build-abi-json.sh",
-        "generate-web3-types": "typechain --target=ethers-v5 --out-dir=./src/typechain \"./src/abi/**/*.json\"",
-        "generate-graphql-types": "graphql-codegen --config subgraph-codegen.yml"
+        "generate:abi-files": "rm -rf src/abi && mkdir src/abi && tasks/build-abi-json.sh",
+        "generate:web3-types": "typechain --target=ethers-v5 --out-dir=./src/typechain \"./src/abi/**/*.json\"",
+        "generate-graphql-schema": "get-graphql-schema https://api.thegraph.com/subgraphs/name/superfluid-finance/protocol-feature-matic > src/subgraph/schema.graphql",
```

Comment:
This is missing ":"

Reply:
no, we don't want to generate this because we still have to go in and manually change stuff in other files.

Reply:
Actually, we don't want to necessarily run this every build so it shouldn't have it...

Reply:
yea, if it is one time during the development, it could be a separate step.

`yarn prepare` for example